data/transactions/logic/assembler_test.go

```diff
@@ -1707,6 +1707,10 @@ block BlkSeed
 global AssetCreateMinBalance
 global AssetOptInMinBalance
 global GenesisHash
+pushint 1
+block BlkProposer
+pushint 1
+block BlkFeesCollected
```

Comment:
True, that was added later. Does anyone have any opinion on whether we should be adding header fields almost by default (as I did for `FeesCollected` and we're contemplating for `Bonus`)?  I feel that `Proposer` is important, meaningful contracts might well be written to do something based on the proposer of a block (most obviously, third party rewards would be possible).  I'm not so sure about FeesCollected or Bonus (though I think they should probably get the same treatment).

Looking for input.

Reply:
I misunderstood your comment, because I didn't remember addign `BlkBonus` as a field. I thought you were asking for it.  I'll fix the test.

But the question remains _should_ we have BlkBonus and BlkFeesCollected? I guess it doesn't hurt.

Reply:
I struggle to think of a legitimate use case that demands those fields, but I certainly don't think they must be hidden.

Exposing new block header fields is _probably_ a good policy, but realistically if there's no demand/use case, it may be a pointless exercise. Anyway, I'm happy with the work done here.